client/components/Main/PreviewHeader.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {EventPreviewHeader} from '../Events';
+import {getItemType} from '../../utils';
+
+export const PreviewHeader = ({item}) => (
+    <div>
```

Comment:
If the item type is `Event`, directly return the `EventPreviewHeader` component (removing the parent div node, as it is not required)